### ISSUE TITLE
Report if [Test] and [TestCase] are used together and provide codefix to remove [Test]

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#2067]
We should report if an NUnit test uses both `[Test]` and `[TestCase]`.
It should either use `[Test]` or `[TestCase]`.

In addition, we should provide a codefix to remove the `[Test]` attribute.

Note:
There are different situations to be aware of.
1. `Test` is part of an attribute list:
   ```c#
   [Test, TestCase("")]
   public void DoSomething() { }
   ```
2. `Test` is in it's own attribute list:
   ```c#
   [Test][TestCase("")]
   public void DoSomething() { }

   [Test]
   [TestCase("")]
   public void DoSomething() { }
   ```
The codefix has to be aware of both situations.
